src/west/manifest.py

```diff
@@ -1836,6 +1836,9 @@ def _load_projects(self, manifest: Dict[str, Any],
         # information about which ones have imports that need to be
         # processed next.
 
+        if 'projects' not in manifest:
+            return
```

Comment:
I think this was a bit of a hack. Dropping this in #825, see commit messages there.